.config/jest-playwright.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  launchBrowserApp: {
+    headless: process.env.HEADLESS !== 'false',
+    dumpio: true,
```

Comment:
looking at the latest build, it looks like Chromium still needs the args, but they were causing WebKit to break. I think we can replace `dumpio` with args. I also created https://github.com/arjun27/playwright-github-actions/issues/4 to track this

```suggestion
    args: process.env.BROWSER === 'chromium' ? ['--no-sandbox', '--disable-setuid-sandbox'] : []
```